src/inventory_ui.cpp

```diff
@@ -299,12 +299,12 @@ std::string enum_to_string<inventory_selector::uimode>( inventory_selector::uimo
 {
     switch( mode ) {
         case inventory_selector::uimode::hierarchy:
-            return "hierarchy";
+            return "Hierarchy";
```

Comment:
This is used for serialization of the enum, and is not marked for translation. You'd better create a separate function for the translated strings.